package-lock.json

```diff
@@ -68604,29 +68811,6 @@
       "dev": true,
       "optional": true
     },
-    "mongodb": {
-      "version": "4.17.1",
```

Comment:
the `package-lock.json` still has this section for mongoose:

```json
    "node_modules/mongoose": {
      "version": "6.12.3",
      "resolved": "https://registry.npmjs.org/mongoose/-/mongoose-6.12.3.tgz",
      "integrity": "sha512-MNJymaaXali7w7rHBxVUoQ3HzHHMk/7I/+yeeoSa4rUzdjZwIWQznBNvVgc0A8ghuJwsuIkb5LyLV6gSjGjWyQ==",
      "dev": true,
      "dependencies": {
        "bson": "^4.7.2",
        "kareem": "2.5.1",
        "mongodb": "4.17.1",
        "mpath": "0.9.0",
        "mquery": "4.0.3",
        "ms": "2.1.3",
        "sift": "16.0.1"
      },
      "engines": {
        "node": ">=12.0.0"
      },
      "funding": {
        "type": "opencollective",
        "url": "https://opencollective.com/mongoose"
      }
    },
```

with dependency on `"mongodb": "4.17.1"`, so not sure why this part is removed from lock file, but it seems wrong and make tests fail for obvious reasons.

Reply:
Yah, this seems just broken. I don't know what renovate is doing here.